storybook/src/components/LinkList/LinkList.docs.mdx

```diff
@@ -21,8 +21,8 @@ Avoid using both semantic and list icons in the same list.
 
 ### Small text
 
-In the [Page Footer](/docs/components-containers-page-footer--docs), we use small text, also for lists.
-Specify this for each item in the list.
+We use a smaller font size for text in a Figure Caption and for metadata like dates or categories.
+When using a list in small text, set the prop for each item in the list.
```

Comment:
If we only use small text for Avatar, Badge, Character Counter, Figure Caption, and lines of metadata, shouldn't we remove this option for LinkList? Same for ordered and unordered lists.

Reply:
As discussed, we'll handle this in a separate ticket